### PR TITLE
test(first-button-test):

### PR DESCRIPTION
**Pull request by Jorchar**
Added first test

### CODE REVIEW

**Review by Dastroo — CHANGES_REQUESTED** [review]

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -20,6 +20,15 @@ describe('AppComponent', () => {
     expect(app.title).toEqual('firstAngularApp');
   });
 
+  it('should button change value', () => {
```
[the remark this answers is not in the document]
Comment (Jorchar) [reply src/app/app.component.spec.ts:23]: It is not button value, but value changed by button

[the remark this answers is not in the document]
Comment (Jorchar) [reply src/app/app.component.spec.ts:23]: What about: 
button should change value
?

Comment (Jorchar) [inline src/app/app.component.spec.ts:23]:
```suggestion
  it('button should change value', () => {
```

[the remark this answers is not in the document]
Comment (Jorchar) [reply src/app/app.component.spec.ts:23]:
```suggestion
  it('button should change value', () => {
```

**Review by Dastroo — APPROVED** [review]
LGTM

**Pull request merged**